setup.cfg

```diff
@@ -4,6 +4,8 @@ version = attr: falcon.__version__
 description = An unladen web framework for building APIs and app backends.
 long_description_content_type = text/x-rst
 url = https://falconframework.org
+project_urls =
+    Source = https://github.com/falconry/falcon
```

Comment:
the project urls already exists. we may want to add that page, but this needs to go in the existing section